### ISSUE TITLE
Can't follow remotely or search federated instances 

### ISSUE THREAD

**Issue by arjonny** [pixelfed/pixelfed#1662]
I am having difficulty federating or connecting to any instances at all, and I am wondering if I am missing something I am running the latest version
and have edited my .env like this

```
ACTIVITY_PUB=true
REMOTE_FOLLOW=true
ACTIVITYPUB_INBOX=true
ACTIVITYPUB_SHAREDINBOX=true
# Set these "true" to enable federation.
# You might need to also run:
#   php artisan cache:clear
#   php artisan optimize:clear
#   php artisan optimize

PF_COSTAR_ENABLED=true
CS_BLOCKED_DOMAINS='gab.com,gab.ai,develop.gab.com'
CS_CW_DOMAINS='switter.at'
CS_UNLISTED_DOMAINS='example.org,example.net,example.com'
```

I am still getting errors trying to follow accounts from different instances, and the search doesn't turn up any results at all when searching users. 

![Screenshot_20190904_153303](https://user-images.githubusercontent.com/29845448/64286982-85b12300-cf2c-11e9-8d5b-c0270bc85587.jpg)

Are there some additional settings I am not aware of?

**Comment by trwnh** [comment]
Not sure about the search not working, but Mastodon's interaction redirect is not supported. Remote follow is currently only possibly by searching for the HTTPS URI of the profile you intend to follow, directly within Pixelfed

**Comment by arjonny** [comment]
Thank you for your reply.
I have narrowed it down to either permissions, or php modules, or something weird with my nginx settings proxying files incorrectly (in other words pretty much anything?)
I am also having an issue with “No Preview Available” in discover and my profile page overview.

I have set everything up I think to the letter of the law from the documentation, the only thing I know I did wrong is I didn’t not have all the php modules enabled during the initial install, whoops.  I enabled them and ran compose again, but is there any chance my instance is broken and needs a fresh start? Or do you suspect this can be fixed?

The other thing I wondered is perhaps my VPS is just not able to handle the processes with limited memory perhaps something is breaking behinds the scenes.  I didn’t see any system requirements so I am using a 1gb ram machine. So perhaps I am running out of memory?

I am extremely new to this, please be patient with my inexperience, I am trying my best to learn.  

![Annotation 2019-09-05 081614](https://user-images.githubusercontent.com/29845448/64342342-4d572680-cfb8-11e9-9457-aa77a506958d.png)

**Comment by raphgilles** [comment]
Just to be sure: did you do `php artisan config:cache` after editing the configuration?

**Comment by arjonny** [comment]
Yes I did, it still returns no results. I have tried just about everything I can think of and even started over entirely a several times. I have probably set up about 8 or 9 PixelFed instances in the past several days and each time I end up with a similar result. Everything is operational except I cant search users from other instances when using the full HTTPS address.

There must be some kind of hidden issue I dont know where to look.
